README.md

```diff
@@ -7,14 +7,6 @@
 
 Gitpay is an open-source platform that empowers collaboration and rewards contributions to open-source projects.
 
-https://gitpitch.com/worknenjoy/gitpay
-
-### Slack channel
-
-We have a slack channel to collaborate with solutions and to help you, and to be fun
-
-[Join us on slack](https://join.slack.com/t/gitpay-workspace/shared_invite/zt-1j8yyotnp-EbA3OSuL7cJkZL_CgBW37g)
-
```

Comment:
The slack channel is working and updated, just need to remove the Gitpitch link

Reply:
When I use this link I get the message "This link is no longer active".

Reply:
Okay, I found a working URL on the website :slightly_smiling_face: